packages/react-form-state/src/tests/FormState.test.tsx

```diff
@@ -105,15 +101,37 @@ describe('<FormState />', () => {
 
       const formDetails = lastCallArgs(renderPropSpy);
       formDetails.fields.product.onChange(faker.commerce.productName());
-      form.update();
 
       const {reset} = lastCallArgs(renderPropSpy);
       reset();
 
-      form.update();
-
       const {fields} = lastCallArgs(renderPropSpy);
+      expect(fields).toMatchObject({
```

Comment:
I left a lot of these using `lastCallArgs` because we only actually care about a few of the fields, and doing  `expect.objectContaining` with `toHaveLastBeenCalledWith` adds too much noise and stuff we don't care about IMO. 

I did use that strategy in places we do actually care about every field though.